src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    private static final FigureSupplier figureSupplies = new FigureSupplier();
```

Comment:
Don't use static methods or fields unless they are constants or utilities. Here, you should create an instance of `FigureSupplier` inside the `main` method rather than having a static field.